<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureNameDialog</class>
 <widget class="QDialog" name="ConfigureNameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configure Name</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string notr="true">TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Move:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>dataEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&amp;JSON string:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>dataEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="dataEdit">
       <property name="toolTip">
        <string>Enter JSON string that will be associated with the name</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="labelJsonValid">
       <property name="text">
        <string notr="true">TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelCurrentAddress">
       <property name="text">
        <string>&amp;Address:</string>
       </property>
       <property name="buddy">
        <cstring>copyButton</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelAddress">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Chrono Kings address to which the name is assigned</string>
         </property>
         <property name="text">
          <string notr="true">CJt898fvQDvjw1SBKqxCCd9hpZd4u1Qsn5</string>
         </property>
         <property name="indent">
          <number>3</number>
         </property>
         <property name="buddy">
          <cstring>transferTo</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="copyButton">
         <property name="toolTip">
          <string>Copy address to clipboard</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../bitcoin.qrc">
           <normaloff>:/icons/editcopy</normaloff>:/icons/editcopy</iconset>
         </property>
         <property name="shortcut">
          <string>Alt+P</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelTransferTo">
       <property name="text">
        <string>&amp;Transfer to:</string>
       </property>
       <property name="buddy">
        <cstring>transferTo</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="transferToLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QValidatedLineEdit" name="transferTo">
         <property name="toolTip">
          <string>The Chrono Kings address to transfer
the domain to, e.g.
CJt898fvQDvjw1SBKqxCCd9hpZd4u1Qsn5

Leave empty, if not needed.</string>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="maxLength">
          <number>34</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addressBookButton">
         <property name="toolTip">
          <string>Choose address from address book</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../bitcoin.qrc">
           <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
         </property>
         <property name="shortcut">
          <string>Alt+A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pasteButton">
         <property name="toolTip">
          <string>Paste address from clipboard</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../bitcoin.qrc">
           <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
         </property>
         <property name="shortcut">
          <string>Alt+P</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="labelTransferToHint">
       <property name="text">
        <string>(can be left empty)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>C&amp;hat message:</string>
         </property>
         <property name="buddy">
          <cstring>messageEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPlainTextEdit" name="messageEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelSpawn">
         <property name="text">
          <string>Spawn:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayoutSpawn">
         <item>
          <widget class="QLabel" name="labelColor">
           <property name="text">
            <string>Select color:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="redButton">
           <property name="text">
            <string>Red</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="blueButton">
           <property name="text">
            <string>Blue</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelStep">
         <property name="text">
          <string>Move:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelAttack">
         <property name="text">
          <string>Attack:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayoutStep">
         <item>
          <widget class="QPushButton" name="upButton">
           <property name="text">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="downButton">
           <property name="text">
            <string>Down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="leftButton">
           <property name="text">
            <string>Left</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rightButton">
           <property name="text">
            <string>Right</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayoutAttack">
         <item>
          <widget class="QLabel" name="labelReachablePlayers">
           <property name="text">
            <string>Reachable players:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxAttack">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelSubmitHint">
       <property name="text">
        <string notr="true">TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../../src/qt/qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>transferTo</tabstop>
  <tabstop>addressBookButton</tabstop>
  <tabstop>pasteButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureNameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureNameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
